libs/counterfactuals/src/util/getCurrentLabel.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export function getCurrentLabel(
+  taskType: "classification" | "regression" | "text_classification",
```

Comment:
is there some way to move these to constants?

Reply:
Updated. Created an enum DatasetTaskType in core-ui. Thanks!